.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,20 +2,20 @@ I want to merge this change because...
 
 <!-- Please mention all relevant issue numbers. -->
 
-### Screenshots
+# Impact
 
-<!-- If your changes affect the UI, providing "before" and "after" screenshots will
-greatly reduce the amount of work needed to review your work. -->
+* [ ] New mutations
```

Comment:
Doesn't the second point cover this one?

Reply:
Sorry, it was supposed to say "migrations"